.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,21 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Simulate Docker Build"
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 2 * * *' # run daily
+
+jobs:
+  docker-build:
+    name: "Simulate Docker Build"
+
+    runs-on: ${{ matrix.operating-system }}
```

Comment:
There is no matrix configured? Why not just `"ubuntu-latest"`?

Reply:
Good point 👍

Reply:
44344879748d773c5945a4508712c806417d0bcb